app/actions/remote/search.ts

```diff
@@ -7,65 +7,73 @@ import NetworkManager from '@managers/network_manager';
 import {prepareMissingChannelsForAllTeams} from '@queries/servers/channel';
 import {getIsCRTEnabled, prepareThreadsFromReceivedPosts} from '@queries/servers/thread';
 import {getCurrentUser} from '@queries/servers/user';
-import {processPostsFetched} from '@utils/post';
 
 import {fetchPostAuthors, fetchMissingChannelsFromPosts} from './post';
 import {forceLogoutIfNecessary} from './session';
 
 import type {Client} from '@client/rest';
 import type Model from '@nozbe/watermelondb/Model';
 
+type FileSearchRequest = {
+    error?: unknown;
+    file_infos?: {[id: string]: FileInfo};
```

Comment:
I'm no liking this type, but we can change whenever we add support for searching files